librz/arch/p/asm/asm_hppa_cs.c

```diff
@@ -0,0 +1,123 @@
+// SPDX-FileCopyrightText: 2024 Anton Kochkov <anton.kochkov@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_asm.h>
+#include <rz_lib.h>
+#include <capstone/capstone.h>
+
+static int disassemble(RzAsm *a, RzAsmOp *op, const ut8 *buf, int len) {
+	if (!buf || !op || !a->plugin_data) {
+		return -1;
+	}
+
+	RzAsmHPPAContext *ctx = a->plugin_data;
+	if (!hppa_setup_cs_handle(ctx, a->cpu, a->features, a->big_endian)) {
+		return -1;
+	}
+
+	op->size = 4;
+
+	ctx->insn = NULL;
+	ctx->count = cs_disasm(ctx->h, buf, len, a->pc, 1, &ctx->insn);
+	if (ctx->count <= 0) {
+		RZ_LOG_ERROR("HPPA: disasm error @ 0x%08" PFMT64x ", len = %d\n", a->pc, len);
```

Comment:
remove this.

Reply:
It's here for debugging. Once PR is ready I will remove.